CoordgenMinimizer.cpp

```diff
@@ -72,7 +74,18 @@ void CoordgenMinimizer::run()
     }
 
     for (int iterations = 0; iterations < m_maxIterations; ++iterations) {
-        scoreInteractions();
+        float energy = scoreInteractions();
+
+        // #ifdef ...
```

Comment:
I'll let Dan decide a name for the macro, I'm just wondering if we want that or an environment variable